sql/moz-fx-data-shared-prod/glean_auto_events_derived/apps_auto_events_metadata_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+friendly_name: Apps Auto Events Metadata
+description: |-
+  Holds apps metadata about the autogenerated events that are collected by Glean.
+owners:
+- efilho@mozilla.com
+labels:
+  incremental: false
+  owner1: efilho
+  public_json: true
+  public_sql: false
+  review_bugs:
+  - '1947481'
+scheduling:
+  dag_name: bqetl_glean_dictionary
+  depends_on_past: false
+  depends_on:
+  - task_id: daily_auto_events_metadata
+    dag_name: bqetl_glean_dictionary
```

Comment:
Do we need to specify this manually? It seems to be causing validate-dags to fail.

Reply:
I don't really know. Let's try without it